roles/openshift_setup/tasks/main.yml

```diff
@@ -167,6 +167,22 @@
             additionalTrustedCA:
               name: "registry-cas"
 
+- name: Create a ICSP with repository digest mirrors
+  when:
+    - cifmw_openshift_setup_digest_mirrors is defined
+    - cifmw_openshift_setup_digest_mirrors | length > 0
```

Comment:
Can we create one of this in the molecule test just to be sure the task executes and it's able to create what we put in the definition?

Reply:
Yes, let me add it.

Reply:
much better now, thanks for the heads up. ++